### ISSUE TITLE
Minilens walk animation seems choppy

### ISSUE THREAD

**Issue by akien-mga** [alketii/minilens#93]
In current master, when you walk in straight line the animation seems choppy (at least on Linux/Kwin with intel drivers).

**Comment by ObaniGemini** [comment]
Mmmmh maybe it came from my last contributions. I tried to make a greater impression of movement to the player, sometimes maybe overdoing it

**Comment by akien-mga** [comment]
Indeed, it seems to be a regression from 6cbc7807. Reverting this commit fixes the issue (but loses many changes of course, so it should not be done :)).

**Comment by ObaniGemini** [comment]
Just say what needs to be reworked

**Comment by akien-mga** [comment]
> when you walk in straight line the animation seems choppy
